gdb/plan.sh

```diff
@@ -1,32 +1,57 @@
 pkg_name=gdb
 pkg_origin=core
-pkg_version=7.11
+pkg_version=7.11.1
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('gplv3')
-pkg_source=http://ftp.gnu.org/gnu/$pkg_name/${pkg_name}-${pkg_version}.tar.xz
-pkg_shasum=7a434116cb630d77bb40776e8f5d3937bed11dea56bafebb4d2bc5dd389fe5c1
-pkg_deps=(core/glibc core/readline core/zlib core/python)
-pkg_build_deps=(core/coreutils core/diffutils core/patch core/make core/gcc core/texinfo)
+pkg_description="GDB, the GNU Project debugger, allows you to see what is going on 'inside' another program while it executes -- or what another program was doing at the moment it crashed."
+pkg_upstream_url=https://www.gnu.org/software/gdb/
+pkg_source=http://ftp.gnu.org/gnu/${pkg_name}/${pkg_name}-${pkg_version}.tar.xz
+pkg_shasum=e9216da4e3755e9f414c1aa0026b626251dfc57ffe572a266e98da4f6988fc70
+pkg_deps=(core/glibc core/readline core/zlib core/expat core/python)
```

Comment:
I tried passing `--enable-tui` and adding `core/ncurses` but it fails to compile, I have to analyze this.